elements/pfe-primary-detail/package.json

```diff
@@ -43,7 +52,14 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@patternfly/pfelement": "1.0.0",
-    "gh-pages": "^2.2.0"
```

Comment:
Gonna assume you meant to remove github pages. 👍

Reply:
Yeah, that's old; we don't use it anymore